deviceModeInit.js

```diff
@@ -171,11 +176,42 @@ var rudderTracking = (function () {
     };
     if (pages[path] === "Registration Viewed") {
       rudderanalytics.track(pages[path], properties);
+      rs$("#create_customer").submit(userRegistered);
+    } else if (pages[path] === "Login Viewed") {
+      rudderanalytics.track(pages[path], properties);
+      rs$("#customer_login").submit(userLoggedIn);
     } else {
       rudderanalytics.page(category, pageName, properties);
     }
   }
 
+
+  function userRegistered() {
+    const email = rs$('#create_customer [type="email"]').val();
+    const firstName = rs$(
+      '#create_customer [name="customer[first_name]"]'
+    ).val();
+    const lastName = rs$('#create_customer [name="customer[last_name]"]').val();
+    rudderanalytics.identify({
+      email,
+      firstName,
+      lastName,
+    });
+  }
+
+  function userLoggedIn() {
+    const email = rs$('#customer_login [type="email"]').val();
+    const firstName = rs$(
+      '#customer_login [name="customer[first_name]"]'
+    ).val();
+    const lastName = rs$('#customer_login [name="customer[last_name]"]').val();
+    rudderanalytics.identify({
+      email,
```

Comment:
User ID and email can be two different values for Shopify users?

Reply:
Yes `userId` is a numeric value collected form `customerId` which is a unique customer identifier for shopify.